### PR TITLE
doc: release notes for nautilus 14.2.2

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Nathan Cutler <ncutler@suse.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: doc/releases/nautilus.rst [hunk 1]
```diff
@@ -1,3 +1,236 @@
...
+  and the new 'ceph osd [un]set-group <flags> <who>' command,
+  which sets flags in batch at the granularity of any crush node,
+  or device class.
+
+* radosgw-admin introduces two subcommands that allow the
+  managing of expire-stale objects that might be left behind after a
+  bucket reshard in earlier versions of RGW. One subcommand lists such
+  objects and the other deletes them. Read the troubleshooting section
+  of the dynamic resharding docs for details.
+
```
Comment (liewegas) [inline doc/releases/nautilus.rst]: I wonder if we should add

```
Known Issues
---------------

* All current Nautilus releases have an issue where deploying a single new (Nautilus) BlueStore OSDs on an upgraded (pre-Nautilus) cluster breaks the pool utilization stats reported by ``ceph df``.  Until all OSDs have been reprovisioned or updated (via ``ceph-bluestore-tool repair``), the pool stats will show values that are lower than the true value.  A fix is in the works but will not appear until 14.2.3.  Users who have upgraded to Nautilus (or considering upgrading) may want to delay provisioning new OSDs until the fix is available in the next release.
```

Comment (smithfarm) [reply doc/releases/nautilus.rst]: added (with some wordsmithing tweaks)

File: doc/releases/nautilus.rst [hunk 2]
```diff
@@ -1,3 +1,247 @@
...
+* cephfs: mds: reset heartbeat during long-running loops in recovery (`issue#40223 <http://tracker.ceph.com/issues/40223>`_, `pr#28611 <https://github.com/ceph/ceph/pull/28611>`_, b'"Yan, Zheng")
+* cephfs: mds: there is an assertion when calling Beacon::shutdown() (`issue#39214 <http://tracker.ceph.com/issues/39214>`_, `issue#38822 <http://tracker.ceph.com/issues/38822>`_, `pr#27852 <https://github.com/ceph/ceph/pull/27852>`_, b'huanwen ren)
+* cephfs: mount: key parsing fail when doing a remount (`issue#40164 <http://tracker.ceph.com/issues/40164>`_, `pr#28610 <https://github.com/ceph/ceph/pull/28610>`_, b'Luis Henriques)
+* cephfs: pybind: added lseek() (`pr#28333 <https://github.com/ceph/ceph/pull/28333>`_, b'Xiaowei Chu)
+* common/assert: include ceph_abort_msg(arg) arg in log output (`pr#27824 <https://github.com/ceph/ceph/pull/27824>`_, b'Sage Weil)
+* common/options: annotate some options; enable some runtime updates (`pr#27818 <https://github.com/ceph/ceph/pull/27818>`_, b'Sage Weil)
+* common/options: update mon_crush_min_required_version=hammer (`pr#27625 <https://github.com/ceph/ceph/pull/27625>`_, b'Sage Weil)
```
Comment (neha-ojha) [inline doc/releases/nautilus.rst]: I see a note about this in 14.2.1 https://github.com/ceph/ceph/pull/27649, did it not make it into 14.2.1 or am I missing something?

**Comment by lordcirth** [comment]
Glad to see progress on 14.2.2! Can anyone say approximately when it will be released? Thanks!

File: doc/releases/nautilus.rst (hunk 2 above)
Comment (liewegas) [reply doc/releases/nautilus.rst]: whoops! i moved the note to 14.2.2

**Comment by liewegas** [comment]
> Glad to see progress on 14.2.2! Can anyone say approximately when it will be released? Thanks!

build should be ready tomorrow

**Pull request merged**